### ISSUE TITLE
General pre-processing before 3-D segmentation

### ISSUE THREAD

**Issue by b2jia** [MouseLand/cellpose#136]
What were some common pre-processing steps the authors took to generate 3-D segmentation data? (ie. Laplacian of Gaussian filter to enhance cell borders?)

I realize every dataset is different (ie. diff morphology, imaging conditions, etc.) but perhaps this is a question on the mind of many users: what are some common signal processing techniques to enhance the input so Cellpose performs better?

Currently the only parameter we are able to tune is the estimated diameter of the cell. I'm finding it rather laborious to correct the masks manually - Cellpose generates many "fragmented" cells for me. 

ex. input: 
![image](https://user-images.githubusercontent.com/37123556/96833379-2e526880-13f5-11eb-84a4-63ef4b06273e.png)

ex. output:

![image](https://user-images.githubusercontent.com/37123556/96833365-2692c400-13f5-11eb-8b1d-0098783e3af8.png)

**Comment by kevinjohncutler** [comment]
I've done some fairly exhaustive experimentation with pre-processing data, and what I found is that it is no replacement at all for training on your own raw, _unprocessed_ data. Even a few images like the one you have shown should be plenty (try it just with one!). That being said, if you only have a few images to segment or aren't set up to make the training data, try a simple image gradient, or the curvefilter() from SuperSegger if you use MATLAB. Those enhance cell boundaries quite nicely and help to normalize the image.

**Comment by b2jia** [comment]
^ This is a great suggestion. I would love to hear when the authors would allow 3-D training (although, I realize that is GPU intensive and may not be practical for the general user).

**Comment by b2jia** [comment]
In case anyone is wondering how they can train 3-D segmentation: you need not - the 2D model can be used to compute flows in XY, XZ and YZ. Indeed, @kevinjohncutler 's suggestion that there is "no replacement at all for training on your own raw, _unprocessed_ data" is true - I get significantly better segmentation after hunkering down, annotating some training data, and training from a pre-trained model:

![image](https://user-images.githubusercontent.com/37123556/97841646-25865000-1c9b-11eb-9b02-9742c7bdc6d8.png)
